packages/office-ui-fabric-react/src/components/Autofill/Autofill.tsx

```diff
@@ -304,6 +304,6 @@ export class Autofill extends BaseComponent<IAutofillProps, IAutofillState> impl
 }
 
 /**
- *  Legacy, @deprecated, do not use.
+ *  Legacy, @deprecated do not use.
```

Comment:
>Legacy,  [](start = 4, length = 8)

Since `@deprecated` is a block tag, it will start a special warning box like seen on this page:  [https://docs.microsoft.com/en-us/javascript/api/sp-http/graphhttpclientresponse?view=sp-typescript-latest](https://docs.microsoft.com/en-us/javascript/api/sp-http/graphhttpclientresponse?view=sp-typescript-latest)

(Note that "This API is now obsolete" is autogenerated text.)

Thus, `@deprecated` should not appear in the middle of a sentence.